.travis.yml

```diff
@@ -22,8 +22,8 @@ jobs:
       env: ARCH=ppc64le LD=ld.lld-11
     - name: "ARCH=s390 BOOT=0"
       env: ARCH=s390
-    - name: "ARCH=x86_64 LD=ld.lld"
-      env: ARCH=x86_64 LD=ld.lld-11
+    - name: "ARCH=x86_64 LD=ld.lld AS=clang"
+      env: ARCH=x86_64 LD=ld.lld-11 AS=clang
```

Comment:
Needs to be `clang-11` due to https://apt.llvm.org binaries.

Follow up thoughts?

1. Swap `AS=clang` with `LD=ld.lld-11` to keep it alphabetical?

2. Should we do a GNU as build on cron?

Reply:
done.